ci/scripts/website_build.sh

```diff
@@ -46,14 +46,17 @@ main() {
         exit 1
     fi
 
-    local -r regex='^([0-9]+\.[0-9]+\.[0-9]+)$'
+    # Docs use the ADBC release so it will just be 12, 13, 14, ...
+    local -r regex='^([0-9]+?)$'
```

Comment:
It seems that we can simplify this.

```suggestion
    local -r regex='^[0-9]+$'
```